index.html

```diff
@@ -24,9 +20,10 @@
             It's currently <span id="screen-size"></span>.
         </strong></p>
         <div class="buttons">
-            <button onclick="startTest()">Start Test</button>
+            <button class="start-tests-button">Start Test</button>
+            <br/>
```

Comment:
I don't think we want to have / in <br> since we're using HTML parser, not XML parser.